.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
```

Comment:
@niyobobo Thank you this is a great approach.

However, I have a few concerns.  You have limited the circle Ci version to 2. I have seen that there is a new version which is stable [2.17.1](https://circleci.com/server/changelog/)

Reply:
Thank you @aaronsekisambu for your feedback. 

In semantic versioning style `[Major.minor.patch]` when you specify the major index it checks the latest stable version, not a fixed version. This means CircleCI will use the latest version started by major two.